scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -391,7 +391,7 @@ static u64 dom_min_vruntime(struct dom_ctx *domc)
 	 *
 	 * XXX: Once those macros are added, update this access.
 	 */
-	return domc->min_vruntime;
+	return READ_ONCE(domc->min_vruntime);
```

Comment:
LGTM, thanks. Could you please remove the comment above as well? I agree that the function doesn't help readability, but it's nice to be able to prevent callers from having to all do `READ_ONCE()` everywhere

Reply:
I see, I've removed the unused comment.